tests/testthat/test-theme_elements_gtsummary.R

```diff
@@ -0,0 +1,212 @@
+# Package-wide Unit Tests-------------------------------------------------------
+## pkgwide-fn:prependpvalue_fun-------------------------------------------------
+test_that("pkgwide-fn:prependpvalue_fun works", {
+
+  # Create a theme#
+  my_theme_1 <-
+    list(
+      # Prepend p value, with 3 place digits
+      "pkgwide-fn:prependpvalue_fun" = label_style_pvalue(digits = 3, prepend_p = TRUE)
+    )
+
+  # Set the theme
+  set_gtsummary_theme(my_theme_1)
+
+  # Store the table#
+  gts_1 <-
+    trial |>
+    tbl_summary(
+      by = trt,
+      include = c(trt, age),
+    ) |>
+    add_p()
+
+  # Test that the p-value has 3 digits#
+  expect_true(
+    inline_text(gts_1, variable = age, column = "p.value") |>
+      grepl(pattern = "p=0.\\d{3}", x = _)
```

Comment:
This will be easier to read as an `expect_equal()`. Can we update to avoid the regex pattern matching?